llvm/lib/Target/RISCV/GISel/RISCVRegisterBanks.td

```diff
@@ -14,3 +14,7 @@ def GPRRegBank : RegisterBank<"GPRB", [GPR]>;
 
 /// Floating Point Registers: F.
 def FPRRegBank : RegisterBank<"FPRB", [FPR64]>;
+
+/// Vector Regististers: V.
+def VRRegBank : RegisterBank<"VRB", [VR]>;
```

Comment:
Please use `VRBRegBank` to match the string "VRB" string passed to the class. I renamed the other two banks over the weekend.